core/modules/layout/layout.install

```diff
@@ -35,6 +35,21 @@ function layout_install() {
   db_query("UPDATE {system} SET weight = 60 WHERE name = 'layout' AND type = 'module'");
 }
 
+/**
+ * Set home page menu to render as a drop-down menu.
+ */
+function layout_update_1000() {
```

Comment:
Let's try putting this update in `standard.install`, as it actually "owns" the layout configuration. Being in layout.install is also correctly breaking the upgrade tests.